doc/xversionmessage.md

```diff
@@ -176,70 +175,23 @@ detailed interpretation of the fields in the `xmap` will obviously
 rely on external specifications that cannot ever be the scope of this
 document.
 
-The `xmap` directory is meant to be machine readable. For this reason,
-a simple `python` tool in the Bitcoin Unlimited implementation
-produces a C++ header file from this directory (see below).
-
-The rationale to not directly implement it as a simple C++ header is
-that this will make it easier for non-C++ implementations to extract
-the desired values from this table as well as ensuring that table
-stays simple.
-
-## `xmap` directory format
-The file is assumed to be ASCII-encoded and read
-line-by-line. Anything that follows a `#` is deemed a comment and
-ignored. Lines that are empty or consisting only of whitespace are
-ignored as well.
-
-Each line is interpreted as a string tuple. The entries in the tuple
-are separated by any positive amount of whitespace in the directory file.
-
-String tuples that have `KEY` as their first entry specify the key type, naming,
-prefix, suffix and data type of an entry in the `xmap`, in this order. For example:
-
-`KEY i BU_LISTEN_PORT                            0x0000                   0x0000         u64c`
-
-
-The name of a key should be an all-upper-case, underscore-separated string
-that is a valid C++ `enum` identifier, like in the above example.
-
-The keys in the table should be marked with either `INITIAL` or
-CHANGEABLE, expressed as `i` or `c`. Currently, only the meaning of
-INITIAL (`i`) is defined.
-
-The idea is that an initial value is marking values that are valid for
-the lifetime of the connection to the peer, whereas values marked as
-`c` could be updated with a to-be-defined update mechanism. This
-functionality, however, has not been implemented yet. And implementer
-that desires the functionality of a `CHANGEABLE` key is expected to
-update this very specification accordingly.
-
-Valid values for the data type specification are currently `vector` for an
-unspecified vector of bytes and `u64c` for a value that is assumed to
-be a compact-size encoded 64-bit unsigned integer like described
-above.
-
-Other tuples that do not start with `KEY` or have the above format are
-currently undefined and should be expected to produce an error in any
-`xmap` directory parsers that might be implemented (or potentially in
-any of the later processing and compilation stages).
-
 ## Implementation prefixes
 
 The implementations listed here should use the following prefixes:
 
-`bitcore           0x0005`
+`ABC               0x0000`
 `bcash             0x0001`
-`Parity Cash       0x0007`
-`bitprim           0x0006`
-`bchd              0x0004`
-`XT                0x0003`
 `BU                0x0002`
-`ABC               0x0000`
+`XT                0x0003`
+`bchd              0x0004`
+`bitcore           0x0005`
+`bitprim           0x0006`
+`Parity Cash       0x0007`
+
```

Comment:
bitprim is no more, @fpelliccioni start working on https://github.com/k-nuth which is a fork of bitprim

unsure about Parity Cash, also bcash seems to be abandoned.

XT is no more either. What's bitcore? 

that said I would add BCN (Bitcoin Cash Node), k-nuth and Bitcoin Verde.

Reply:
Ah yes, i hadnt updated the list, just put it in order by prefix value. bitcore is btc core afaik, if thats not it ud have to ask awemany. 

Will add those 3 you mentioned. That being said i dont think there is currently anything in the spec with what to do regarding implementations that were assigned a prefix but have since been abandoned. I will add a small section for this as well